zign/api.py

```diff
@@ -227,9 +227,15 @@ def get_token_implicit_flow(name=None, authorize_url=None, token_url=None, clien
 
     data = load_config_ztoken(REFRESH_TOKEN_FILE_PATH)
 
-    # Always try with refresh token first
+    # Force prompting for authorize-url and token-url if only one is specified in the parameter list.
+    if authorize_url and not token_url:
```

Comment:
it looks like checking for these parameters is happening too many times in this method. Is there maybe a way to "refactor" this so that the checks do not have to be duplicated?

useRefresh := not authorize_url and not token_url